streamwebs_frontend/streamwebs/templates/streamwebs/admin/user_promo.html

```diff
@@ -0,0 +1,56 @@
+{% extends 'streamwebs/base.html' %}
+{% load i18n %}
+{% load staticfiles %}
+
+{% block title %} Manage User Permissions {% endblock %}
+{% block body_title %} Manage User Permissions {% endblock %}
+
+{% block content %}
+<div class="container">
+{% if msgs %}
+  {% for message in msgs %}
+  <strong>{{ message }}</strong><br>
+  {% endfor %}
+{% endif %}
+
+  <form id='promo_form' method='post'>
+    {% csrf_token %}
+    {{ promo_form.as_p }}
+    <input class="btn waves-effect waves-light teal darken-4" type='submit' name='submit' value='{% trans 'Apply' %}'/>
+  </form>
+
+  {% for user in page_of_users %}
```

Comment:
The user list is a little strange looking, can that be formatted a bit? It looks a little like debug output, it would look better in a table with a header, so the viewer know what it means